src/Raven.Studio/typescript/components/pages/resources/databases/partials/DatabasePanel.tsx

```diff
@@ -148,6 +149,13 @@ export function DatabasePanel(props: DatabasePanelProps) {
 
     const canRestartDatabase = hasDatabaseAdminAccess && !db.isDisabled;
 
+    const hasAnyDropdownActions =
+        canPauseAnyIndexing ||
+        canResumeAnyPausedIndexing ||
+        canDisableIndexing ||
+        canEnableIndexing ||
+        isOperatorOrAbove;
```

Comment:
It's true when isOperatorOrAbove is true, but we want to disable it when db is disabled